docs/platform/advanced-targeting/custom-properties.md

```diff
@@ -144,3 +144,34 @@ You can target users in numerous ways using Custom Properties. The following lis
 **Special Users**. Sometimes organizations want to release a feature to special users only, such as users with a paid membership, those with a free trial, or those who have made large contributions to the company. In this case, some property name suggestions are `accountType` `pricingPlan` `isSubscriber` `isTrialUser` `amountContributed` etc.
 
 **User Behaviour or Preferences.** You can experiment on users based on their behaviour or preferences. Some examples include `numberOfPageVisits` `gaveConsent` `preferredColor` etc.
+
+## Custom Property Schemas
+
+Similar to [Variable Schemas](/platform/advanced-variables/variable-schemas.md), Custom Property Schemas allow you to set up a schema and define a list of possible values for a custom property, which will appear in a dropdown when setting up Targeting Rules. Labels can be defined for each value to make selections more human-readable and easier to identify. This removes the need to remember valid inputs for a specific custom property.
```

Comment:
maybe include an image of what the targeting rules dropdown looks like here?

Reply:
@ajwootto hmm.. i did?

Reply:
oh okay, i cant see the preview :)